src/main/java/frc/robot/commands/IntakeCommand.java

```diff
@@ -0,0 +1,40 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.IntakeSubsystem;
+
+
+public class IntakeCommand extends CommandBase {
+    
+    private IntakeSubsystem intakeSubsystem;
+    
+    public IntakeCommand() {
+        intakeSubsystem = IntakeSubsystem.getInstance();
+        addRequirements(intakeSubsystem);
+        
+        // Initialize the subsystem with this condition
+        intakeSubsystem.setCondition("button", false);

```

Comment:
I wonder if there's a condition label that makes this a bit more clear, such as "activated" or "buttonPressed".

Could this same command theoretically also be used in autonomous or would there need to be a separate AutonomousIntake command?